src/routes/tools/_common/constants.js

```diff
@@ -100,10 +100,10 @@ export const OBSERVED = [
 export const OBSERVED_FILTER_YEAR = 2006;
 
 export const INITIAL_CONFIG = {
-  boundaryId: "locagrid",
-  scenarioId: "rcp45",
-  climvarId: "tasmax",
-  modelIds: ["HadGEM2-ES", "CNRM-CM5", "CanESM2", "MIROC5"],
+  boundary: "locagrid",
```

Comment:
Changed these property names to match what's used for the share link. This makes it easier to overwrite any properties from the share link.